packages/e2e-tests/specs/editor/blocks/navigation.test.js

```diff
@@ -0,0 +1,34 @@
+
+/**
+ * WordPress dependencies
+ */
+import {
+	createNewPost,
+	insertBlock,
+} from '@wordpress/e2e-test-utils';
+
+describe( 'Adds Navigation links', () => {
+	beforeEach( async () => {
+		await createNewPost();
+	} );
+
+	it( 'Should add a link with one click', async () => {
+		const navigationAppender = '.wp-block-navigation .block-list-appender';
+		const emptyOption = '.wp-block-navigation-placeholder__button.is-link';
+		const linkInput = '.block-editor-link-control__search-input';
+		const navigationLink = '.wp-block-navigation-link';
+
+		await insertBlock( 'Navigation' );
+
+		await page.waitForSelector( emptyOption );
```

Comment:
Doesn't look like there's a delay in between inserting a Navigation block and the 'Create empty' button appearing, so I don't think this `waitForSelector` is necessary?